sdk/formrecognizer/Azure.AI.FormRecognizer/tests/DocumentModelAdministrationClient/DocumentModelAdministrationClientLiveTests.cs

```diff
@@ -62,14 +62,45 @@ public async Task StartBuildModel(bool singlePage)
             var trainingFilesUri = new Uri(singlePage ? TestEnvironment.BlobContainerSasUrl : TestEnvironment.MultipageBlobContainerSasUrl);
             var modelId = Recording.GenerateId();
 
-            BuildModelOperation operation = await client.StartBuildModelAsync(trainingFilesUri, modelId);
+            BuildModelOperation operation = await client.StartBuildModelAsync(trainingFilesUri, DocumentBuildMode.Template, modelId);
             await operation.WaitForCompletionAsync();
 
             Assert.IsTrue(operation.HasValue);
 
             DocumentModel model = operation.Value;
 
             ValidateDocumentModel(model);
+
+            Assert.AreEqual(1, model.DocTypes.Count);
+            Assert.IsTrue(model.DocTypes.ContainsKey(modelId));
+
+            DocTypeInfo docType = model.DocTypes[modelId];
+
+            Assert.AreEqual(DocumentBuildMode.Template, docType.BuildMode);
+        }
+
+        [RecordedTest]
+        public async Task StartBuildModelWithNeuralBuildMode()
```

Comment:
Takes around 12~15 min to run live. Less than 1s in playback mode.

Reply:
which training set does this build the model on?

Reply:
It uses the `trainingdata-v3` container. The service has not shared any training set to be used with neural yet, correct?

Reply:
I haven't seen anything, but also not sure we've asked them specifically for a training set.